Project_1/Styles.css

```diff
@@ -1,76 +1,150 @@
-img {
-    src: "Octopus.jpg";
-    max-width: 200%;
-    height: 700px;
-    width: 110%;
-    float: left;
+/* ==========
+GENERAL
+=============*/
+
+*:before,
+*:after {
+    Box-sizing: Border-box;
 }
 
 body {
     background-color: whitesmoke;
 }
 
+img {
+    max-width: 100%;
+}
+
+/* ===========
+TEXT STYLES 
+==============*/
+
+header {
+    border-bottom: 10px solid #018a80;
+    border-top: 10px solid #018a80;
+    margin: 50px 0 100px;
+}
+
 h1 {
-    font-size: 100px;
-    color: #d07f7d;
-    margin-right: 300px;
-    margin-left: 650px;
-    line-height: 1.3;
+    color: #a72323;
+    font-family: 'Noto Sans', sans-serif;
```

Comment:
Note that you call a very specific font here, but if a user doesn't have Noto Sans on their computer, they'll just get a generic sans-serif font.

You should add [Noto Sans from Google Fonts](https://fonts.google.com/specimen/Noto+Sans) and reference their stylesheet in your `index.html` so more users get this font.